scripts/run_server.py

```diff
@@ -21,6 +20,11 @@
 FLAGS = flags.FLAGS
 
 
+def set_logging_and_run(func):
```

Comment:
this looks like a decorator. but i would not turn it into one.
i think things would be more clear to have a run_one_server that takes a cls and a config:
```
def run_one_server(cls, conf):
  logging.set_verbosity(logging.INFO)
  cls(conf).run()
```

then pass args to the Process (see https://docs.python.org/3.4/library/multiprocessing.html?highlight=process#the-process-class)